cluster-up/cluster/kind-k8s-sriov-1.14.2/provider.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -e
+
+
+source ${KUBEVIRTCI_PATH}/cluster/ephemeral-provider-common.sh
```

Comment:
why do we need to source the `ephemeral-provider-common.sh` file here? I wouldn't think there's anything in there to be reused.